data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -869,6 +869,24 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/filter_condition]
 
         {TALK_ABOUT Thrag ( _ "You are close to killing their leader! The unit who finishes him will gain 16 experience points because he is second level. Choose your attacking unit carefully!")}
+
+        [message]
+            speaker=student
+            message= _ "Should I choose a unit that will level up?"
+            message_female= _ "female^Should I choose a unit that will level up?"
+        [/message]
+
+        {TALK_ABOUT Thrag ( _ "That’s a difficult question. Consider which advancements will strengthen your army, and whether the unit should advance now or later.")}
```

Comment:
```suggestion
        {TALK_ABOUT Thrag ( _ "That’s a difficult question. Consider which advancements will strengthen your army the most and whether you need the unit to advance now or if you can wait until later.")}
```

Reply:
Here the "should ... now or later" vs "need ... now or if you can wait" feels like subtly different meanings to me. I think the "should" wording is a better fit, because both options have their own advantages; you might want to wait for the advantage of later healing, even if plenty of xp was available in the current scenario.

The "the most" doesn't need necessary to me.